aiohttp/client_exceptions.py

```diff
@@ -152,6 +152,27 @@ class ClientProxyConnectionError(ClientConnectorError):
     """
 
 
+class UnixClientConnectorError(ClientConnectorError):
+    """Unix connector error.
+
+    Raised in :class:`aiohttp.connector.UnixConnector` if
+        connection to unix socket can not be established.
```

Comment:
Not sure if you can omit `:py:` here. Also, indentation within the same paragraph is rather unexpected.
```suggestion
    Raised in :py:class:`aiohttp.connector.UnixConnector`
    if connection to unix socket can not be established.
```

Reply:
We don't use autodoc for documentation generation; docstrings should be plain texts.